tox.ini

```diff
@@ -2,17 +2,18 @@
 addopts=--tb=short
 
 [tox]
-envlist = lint,isort-check,isort-fix,black-check,black-fix,py36,docs
+envlist = lint,isort-check,isort-fix,black-check,black-fix,py35,py36,py37,docs
 skipsdist = true
 
 [testenv]
 passenv = *
 commands =
+    pip install -e .
     py.test tests --cov-report=term-missing --cov-append --cov=settings_vial {posargs}
     # due to https://github.com/pytest-dev/pytest-cov/issues/175 pytest-cov generates sonar qube incompatible reports
     coverage xml -i
 deps =
-    -rrequirements/requirements-base.txt
+    # -rrequirements/requirements-base.txt
```

Comment:
b/c empty?

Reply:
Yeah, it was throwing some errors with tox. For now this will be empty, so maybe we should remove this line + the file 🤔